metricbeat/module/coredns/_meta/kibana/7/dashboard/Metricbeat-coredns-overview.json

```diff
@@ -0,0 +1,1395 @@
+                  "type": "linear"
+                },
+                "show": true,
+                "style": {},
+                "title": {},
+                "type": "category"
+              }
+            ],
+            "grid": {
+              "categoryLines": false,
+              "style": {
+                "color": "#eee"
+              }
+            },
+            "legendPosition": "right",
+            "seriesParams": [
+              {
+                "data": {
+                  "id": "1",
+                  "label": "Average coredns.stats.dns.response.rcode.count"
+                },
+                "drawLinesBetweenPoints": true,
+                "interpolate": "linear",
+                "lineWidth": null,
+                "mode": "normal",
+                "show": "true",
+                "showCircles": true,
+                "type": "line",
+                "valueAxis": "ValueAxis-1"
+              }
+            ],
+            "times": [],
+            "type": "line",
+            "valueAxes": [
+              {
+                "id": "ValueAxis-1",
+                "labels": {
+                  "filter": false,
+                  "rotate": 0,
+                  "show": true,
+                  "truncate": 100
+                },
+                "name": "LeftAxis-1",
+                "position": "left",
+                "scale": {
+                  "mode": "normal",
+                  "type": "linear"
+                },
+                "show": true,
+                "style": {},
+                "title": {
+                  "text": "Average coredns.stats.dns.response.rcode.count"
+                },
+                "type": "value"
+              }
+            ]
+          },
+          "title": "Responses by Rcode [Metricbeat CoreDNS] ECS",
+          "type": "line"
+        }
+      },
+      "id": "27da53f0-53d5-11e9-b466-9be470bbd327-ecs",
+      "type": "visualization",
+      "updated_at": "2019-04-25T22:47:01.782Z",
+      "version": "WzIzLDFd"
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [
+              {
+                "$state": {
+                  "store": "appState"
+                },
+                "exists": {
+                  "field": "coredns.stats.dns.request.count"
+                },
+                "meta": {
+                  "alias": null,
+                  "disabled": false,
+                  "index": "metricbeat-*",
+                  "key": "coredns.stats.dns.request.count",
+                  "negate": false,
+                  "type": "exists",
+                  "value": "exists"
+                }
+              }
+            ],
+            "index": "metricbeat-*",
+            "query": {
+              "language": "lucene",
+              "query": ""
+            }
+          }
+        },
+        "title": "Requests by Protocol [Metricbeat CoreDNS] ECS",
```

Comment:
This visualization is counting documents.
By what I can read the data comes by `family` `proto` `server` `zone`, and the field contains the total count of requests.

The pie should
- show the number (top hit I guess, since this is incremental) instead of the count of docs
- the number at the metrics endpoint is divided into server/proto/zone/family. We need to account for that, I tested aggregations bucketing on zone, then on protocol, and the 2 concentric circles look much more accurate. Then there are families (I haven't tested ipv6, can we stick for now to family=1 and not support ipv6?). Probably we can split charts by `server` in case someone configures more than one

WDYT?